File: java64/nivel4/exer32.java
import javax.swing.JOptionPane;

public class exer32 {
    public static void main(String[] args) {
        int valor, soma = 0;

        do {
            valor = Integer.parseInt(JOptionPane.showInputDialog("Informe o valor: "));
            for (int i = valor; i < 40 + valor; i++) {
                if (i % 2 == 1) {
                    soma += Math.pow(i, 2);
                }
            }
            JOptionPane.showMessageDialog(null,
                    "O resultado da soma dos quadrados dos 20 primeiros números inteiros positivos ímpares a partir do número informado é:"
                            + soma);
            break;
        } while (valor >= 0 && valor <= 10);
    }
}

File: pyhton64/nivel4/exer32.py
valor = int(input('Informe um valor: '))
soma = 0
if valor > 0 and valor < 10:
    for i in range(valor, 40+valor):
        if i % 2 == 1:
            soma += (2 ** i)
    print(f'Soma dos impares positivos = {soma}')
else:
    print('Valor deve ser maior que 0 e menor que 10')

# Escreva  um  programa  que  calcule  e  exiba  a  soma  dos  quadrados  dos  20
#   primeiros  números  inteiros  positivos 
# ímpares a partir do número informado pelo usuário menor que 10 e maior que zero